runhouse/resources/serverless/aws_lambda_function.py

```diff
@@ -320,7 +323,6 @@ def _create_layer_zip(self):
             shutil.rmtree(dir_name)
             return self.EMPTY_ZIP
```

Comment:
instead of having a separate class attribute to mange this state, just return None?

Reply:
makes sense, changed.